### PR TITLE
Minor typo... "CoOrd" should be "Coord".

### PR DESCRIPTION
**Pull request by microsmith**
This:
  "This allows CoOrd objects to be created with default or specific initial values, like this:"
should be:
  "This allows Coord objects to be created with default or specific initial values, like this:"

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/csharp/programming-guide/classes-and-structs/instance-constructors.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Instance constructors are used to create and initialize any instance member vari
   
  [!code-csharp[csProgGuideObjects#76](../../../csharp/programming-guide/classes-and-structs/codesnippet/CSharp/instance-constructors_2.cs)]  
   
- This allows `CoOrd` objects to be created with default or specific initial values, like this:  
+ This allows `Coord` objects to be created with default or specific initial values, like this:
```
Comment (pkulikov) [inline docs/csharp/programming-guide/classes-and-structs/instance-constructors.md]: According to the published code
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/instance-constructors

it should end with "s": `Coords`

**Review by BillWagner — APPROVED** [review]
Thanks for spotting this and submitting the fix @microsmith 

I've approved this after accepting @pkulikov 's suggestion, and I'll :shipit: now.

Thanks for making your first contribution to docs! 🎆

**Comment by microsmith** [comment]
> 
> 
> Thanks for spotting this and submitting the fix @microsmith
> 
> I've approved this after accepting @pkulikov 's suggestion, and I'll :shipit: now.
> 
> Thanks for making your first contribution to docs! fireworks

Actually, I've made several. This may be the first one in the .NET area.

Question... Is there a link somewhere here in GitHub that links back to the article/page this edit is about?

Thanks,
Mke

**Comment by BillWagner** [comment]
@microsmith Usually we use hackable URLs, and the live URL maps closely to the document in the repository.  For instance, this PR changes `https://github.com/dotnet/docs/blob/master/docs/csharp/programming-guide/classes-and-structs/instance-constructors.md` which maps to `https://docs.microsoft.com/dotnet/csharp/programming-guide/classes-and-structs/instance-constructors`.  In general, replace `https://github.com/dotnet/docs/blob/master/docs/`with `https://docs.microsoft.com/dotnet/`

**Pull request merged**